### ISSUE TITLE
[Accessibility]: Accessible name does not match or contain the visible label text

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2196]
## Detailed description

> Describe in detail the issue you're having.

This issue is specific to one of the `<button>` elements used to create header inside the `<ibm-table>` component, where accessible name does not match or contain the visible label text

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-table>`

> What did you expect to happen? What happened instead? What would you like to see changed?

Accessible name must match or contain the visible label text

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 2.5.3 Label in Name

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178187027-aa2c7b00-9ffd-4617-ab52-02998f9e6cde.png)

**Comment by Akshat55** [comment]
Could you please recreate this in code sandbox? 

I understand that this issue is part of an element in Table heading `<th>`, but other then that I don't know where to look when you state `specific to one `<button>` element`.

Have you tried checking the content of the button? The plugin gives the element location, could you please paste that.

**Comment by Akshat55** [comment]
Closing this issue due to inactivity. This issue does not provide enough information to address & it appears that a custom template is being used in the headers.
